### PR TITLE
Fix relative file paths plus sequence_models and ESM dependency issues

### PR DESCRIPTION
**Pull request by kevingreenman**
I'm suggesting a few fixes to issues I came across while trying to run the `baselines/embeddings/embeddings.py` script:

- relative file paths: several file paths were incorrect
- `sequence_models` not listed as required package in baselines environment yaml file ( #8 )
- the ESM `extract.py` script is used in `embeddings.py`, so I made ESM a submodule of FLIP and included a reminder message to the user to init/update the submodule if they get a `FileNotFoundError` while trying to execute that part of the script

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "baselines/esm"]
+	path = baselines/esm
+	url = git@github.com:facebookresearch/esm.git
```
Comment (sacdallago) [inline .gitmodules:3]: I'd pin the specific version/commit, given that things may change ;)

File: baselines/embeddings/embeddings.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 import argparse 
 
-sys.path.append('/../../FLIP/baselines/')
+sys.path.append('../../FLIP/baselines/')
```
Comment (sacdallago) [inline baselines/embeddings/embeddings.py:19]: Good point

**Review by sacdallago — APPROVED** [review]
Hi @kevingreenman , thanks for the PR. It generally looks good to me.

Interestingly, I've started building out an alternative to the training part of FLIP (not public yet). I thought it'd be better to separate data vs. training part out (and execution: at the end, people may be interested in running predictions!). Would you be interested in having a look? What are you using FLIP for?

File: .gitmodules (hunk 1 above)
Comment (kevingreenman) [reply .gitmodules:3]: good point, I just added a version tag to this

**Comment by sacdallago** [comment]
Oh, so you chat with Kevin already :) Cool -- I suggest we set up a call? Then I can run you through things, in the meantime I'll add you to the repo

**Comment by kevingreenman** [comment]
Thanks! Joined your other repo today. Can you email me at kpg [at] mit [dot] edu to set up a time to meet?

**Pull request merged**